vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +104,128 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
         return aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
     }
     
+    /**
+     * This method maps the title in ExhibitionAboutPage, and add that to
+     * exhibitionTitles list for each user selected Exhibition Language.
+    */
+    public void setAboutPageTitle(LocalizedTextForm title, ExhibitionAboutPage exhibitionAboutPage) {
+        if(title!=null) {
+            
+            LocalizedText localizedText = localizedTextRepo.findById(title.getLocalisedTextId()).orElse(null);
+            if(localizedText != null) {            
+                localizedText.setText(title.getText());            
+            } else {
+                
+                ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findById(title.getExhibitionLanguageId()).orElse(null);
+                if(exhibitionLanguage != null) {
+                    localizedText = new LocalizedText(exhibitionLanguage, title.getText());
+                    exhibitionAboutPage.getExhibitionTitles().add(localizedText);
+                    exhibitionLanguage.getLocalizedTexts().add(localizedText);
```

Comment:
is there ever a need to get all texts from a language? if not, I would not bother with a localizedTexts property.